### PR TITLE
Update tables.md with sql column names

### PR DESCRIPTION
**Pull request by t-young31**
Runs the latest version of the dd_extractor to create tables.md with lowercase and underscored columns names. 

***
### Questions:

- Do we want the audit entries in?
> No

- Should this be rebased and merged directly into main
> Yes

### CODE REVIEW

**Review by AnikaC-git — APPROVED** [review]
That looks good to me and thanks ever so much for changing this over! 🥳 

I would say no to the audit tables as it might confuse users how to use them properly. I see them as something for us to work out problems when things go wrong with processing a specific data type.

File: data_dictionary/tables.md [hunk 1]
```diff
@@ -59,20 +105,43 @@ Holds information relevant to consultation requests for patients.
...
+| consultation_request_id | bigint | Unique identifier in EMAP for this **ConsultationRequest** record. |
+| consultation_type_id | [ConsultationType](#ConsultationType) | Identifier for the [ConsultationType](#ConsultationType) associated with this record. |
+| hospital_visit_id | [HospitalVisit](#HospitalVisit) | Identifier for the [HospitalVisit](#HospitalVisit) associated with this record. |
+| internal_id | bigint | Identifier used in source system for this **ConsultationRequest**. |
+| closed_due_to_discharge | boolean | Predicate determining whether this **ConsultationRequest** was closed on discharge. |
 | comments | varchar(255) | Notes added to the **ConsultationRequest** which are not tied to a Question. |
-| statusChangeTime | timestamp without timezone | Date and time at which this **ConsultationRequest** was last updated. |
-| scheduledDatetime | timestamp without timezone | Date and time at which this **ConsultationRequest** was scheduled. |
+| status_change_time | timestamp without timezone | Date and time at which this **ConsultationRequest** was last updated. |
```
Comment (AnikaC-git) [inline data_dictionary/tables.md:114]: should that not be status_change_datetime? or is that not come through yet on develop?

File: data_dictionary/tables.md [hunk 2]
```diff
@@ -205,9 +366,47 @@ This table stores a mapping from every MRN known to the system, to its currently
...
+| mrn_to_live_id | bigint | Unique identifier in EMAP for this **MrnToLive** record. |
+| mrn_id | [Mrn](#Mrn) | Identifier for the [Mrn](#Mrn) associated with this record. |
+| live_mrn_id | [Mrn](#Mrn) | Identifier for the [Mrn](#Mrn) associated with this record. |
+
+---
+
+
+
+## MrnToLiveAudit
+
```
Comment (AnikaC-git) [inline data_dictionary/tables.md:378]: Probably shouldn't be here

File: data_dictionary/tables.md [hunk 3]
```diff
@@ -205,9 +366,47 @@ This table stores a mapping from every MRN known to the system, to its currently
...
+| mrn_id | [uk.ac.ucl.rits.inform.informdb.identity.Mrn](#uk.ac.ucl.rits.inform.informdb.identity.Mrn) | None |
+| live_mrn_id | [uk.ac.ucl.rits.inform.informdb.identity.Mrn](#uk.ac.ucl.rits.inform.informdb.identity.Mrn) | None |
+
+---
+
+
+
+## ClimbSequenceAudit
+
+Audit table of ClimbSequence.
```
Comment (AnikaC-git) [inline data_dictionary/tables.md:396]: Probably shouldn't be here

File: data_dictionary/tables.md [hunk 4]
```diff
@@ -221,11 +420,30 @@ This represents all the different batteries of test that can be ordered.
...
 | description | varchar(255) | Desription of this battery of tests. |
-| labProvider | varchar(255) | Source system of this batteryCode. |
+| lab_provider | varchar(255) | Source system of this batteryCode. |
+
+---
+
+
+
+## LabBatteryAudit
+
```
Comment (AnikaC-git) [inline data_dictionary/tables.md:434]: Probably shouldn't be here

File: data_dictionary/tables.md [hunk 5]
```diff
@@ -347,30 +678,30 @@ Sensitivities show the affect of specific agents on isolates from cultures.
...
 | sensitivity | varchar(255) | Sensitivity of the microbe to the agent. |
-| reportingDatetime | timestamp without timezone | Date and time at which this **LabSensitivity** was reported. |
+| reporting_datetime | timestamp without timezone | Date and time at which this **LabSensitivity** was reported. |
 
 ---
 
 
 
-## LabTestDefinition
+## LabSensitivityAudit
```
Comment (AnikaC-git) [inline data_dictionary/tables.md:691]: Probably shouldn't be here

File: data_dictionary/tables.md [hunk 6]
```diff
@@ -487,14 +818,36 @@ This represents a patient being in a location for an amount of time.
...
+| location_id | [Location](#Location) | Identifier of the [Location](#Location) associated with this **LocationVisit**. |
+| inferred_admission | boolean | Predicate determining whether the admission time has been inferred (not set from an A01, A02 or A03). |
+| inferred_discharge | boolean | Predicate determining whether discharge time has been inferred (not set from an A01, A02 or A03). |
+
+---
+
+
+
+## LocationVisitAudit
+
```
Comment (AnikaC-git) [inline data_dictionary/tables.md:835]: Probably shouldn't be here

File: data_dictionary/tables.md [hunk 7]
```diff
@@ -601,16 +997,41 @@ VisitObservationType describes the meaning behind a specific observation.
...
+| primary_data_type | varchar(255) | Data type expected to be returned. |
+| creation_time | timestamp without timezone | Date and time at which this **VisitObservationType** was created in the source system. |
+
+---
+
+
+
+## VisitObservationTypeAudit
+
+Audit table of VisitObservationType.
```
Comment (AnikaC-git) [inline data_dictionary/tables.md:1018]: Probably shouldn't be here

**Comment by AnikaC-git** [comment]
@stefpiatek @skeating  do we still keep [this data dictionary](https://github.com/inform-health-informatics/internal_emap_docs/blob/main/data_dictionary/star_data.md) updated? It looks like a very, very early version of the export?

File: data_dictionary/tables.md (hunk 1 above)
Comment (t-young31) [reply data_dictionary/tables.md:114]: It's not been merged into master and I anticipate this PR going straight to main? As to reflect the current state of  the live DB

> NOTE: the changed tables.md was generated from the latest release tag

Comment (stefpiatek) [reply data_dictionary/tables.md:68]: doesn't really make a difference at the moment as I think we'll be releasing soon, but this should reflect what is currently in `master` (and released to the users!), not what's in develop

**Review by stefpiatek — APPROVED** [review]
Looks good to me, though I think we've been a bit keen and created this from develop instead of main?

**Comment by stefpiatek** [comment]
actually ignore me, not sure where I thought I saw a `time` -> `datetime` and had assumed it was because we were on develop

**Pull request merged**